noah/src/main/scala/edu/uci/ics/cloudberry/noah/kafka/ConsumerHistUsersTimeline.scala

```diff
@@ -0,0 +1,27 @@
+package edu.uci.ics.cloudberry.noah.kafka
+
+import edu.uci.ics.cloudberry.noah.feed.{CmdLineAux, Config}
+import org.kohsuke.args4j.CmdLineException
+
+/**
+  * Created by Monique on 7/18/2016.
+  */
+
+object ConsumerHistUsersTimeline {
```

Comment:
I think there are three `main` functions that are sharing the very similar code. you could make a general class and extend it for the main function.